pkg/daemon/controller/controller.go

```diff
@@ -352,6 +354,29 @@ func (sc *ServiceController) expectedServiceCommand(expected servicescm.Service)
 	for key, value := range psVars {
 		expectedCmd = strings.ReplaceAll(expectedCmd, key, value)
 	}
+	if strings.Contains(expectedCmd, services.NodeIPVar) {
```

Comment:
Please mention that this goes against the WICD design principle add a link to a Jira story where you will address this.